src/nimblepkg/options.nim

```diff
@@ -386,10 +395,11 @@ proc parseFlag*(flag, val: string, result: var Options, kind = cmdLongOption) =
     result.showHelp = false
     result.setRunOptions(flag, getFlagString(kind, flag, val), false)
   of actionCustom:
-    if result.action.command.normalize == "test":
-      if f == "continue" or f == "c":
+    if result.action.command.normalize == "test" and
+      f == "continue" or f == "c":
         result.continueTestsOnFailure = true
-    result.action.flags[flag] = val
+    elif not isGlobalFlag:
+      result.setCustomOptions(getFlagString(kind, flag, val))
```

Comment:
This means that a `custom` `test` task will not get a `--continue`/`-c` flag. I would consider that a bug.

Reply:
Why wouldn't it? It would get handled in the previous if condition right?

Reply:
The custom task won't be able to access the flags because they are not added to the "custom options". The only way they would be able to read it is if you make ``continueTestsOnFailure`` accessible. See what I mean?

Reply:
Ah you mean to forward that as a run flag in the new design. I'll clean it up accordingly.

Reply:
> Also, I'm a little concerned about just passing any unknown flag to Nim. Could we whitelist the flags instead of just passing any unknown flags to Nim?

Nim complains if it gets invalid flags. And I'd rather not maintain a list since Nim is a moving target of flags.

Reply:
Fixed.